data/lists/cybralist.com.json

```
@@ -1,29 +1,29 @@
 {
     "$schema": "../../schema/list.json",
-    "id": "bhlist.co.in",
+    "id": "cybralist.com",
     "added": 1660331331,
-    "name": "BHList",
-    "url": "https://bhlist.co.in/",
+    "name": "Cybralist",
+    "url": "https://cybralist.com/",
     "icon": "https://cdn.bhlist.co.in/assets/logo.png",
     "language": "English",
     "display": 1,
     "defunct": 0,
     "discord_only": 1,
     "description": "Spice up your Discord experience with our diverse range of Discord bots",
     "api_docs": "https://docs.bhlist.co.in/",
-    "api_post": "https://api.bhlist.co.in/post/stats",
+    "api_post": "https://api.cybralist.com/post/stats",
     "api_post_method": null,
     "api_field": "server_count",
     "api_shard_id": null,
     "api_shard_count": "shard_count",
     "api_shards": null,
-    "api_get": "https://api.bhlist.co.in/bots/:id",
+    "api_get": "https://api.cybralist.com/bots/:id",
     "api_all": null,
-    "view_bot": "https://bhlist.co.in/bots/:id",
-    "bot_widget": "https://bhlist.co.in/bots/:id/embed",
+    "view_bot": "https://cybralist.com/bots/:id",
+    "bot_widget": "https://api.cybralist.com/bots/:id/embed",
     "content": null,
-    "owners": "{Ansh}#0607 (671390595184459782), abubakar#0834 (1150257050778087435)",
-    "discord": "https://discord.gg/xtx49HtRMb",
+    "owners": "cybrancee (327075863957078026)",
```

```suggestion
    "owners": "cybrancee#0000 (327075863957078026)",
```
For users with a migrated username, the formatting will expect a 0000 discrim